src/bidiMapper/bidiMapper.ts

```diff
@@ -20,7 +20,5 @@
  * Other modules should only access exports defined in this file.
  * XXX: eslint rule for this.
  */
-export {CdpConnection, CdpClient} from './CdpConnection.js';
-export {BidiServer} from './BidiServer.js';
-export {BidiTransport} from './BidiTransport.js';
-export {EventEmitter} from '../utils/EventEmitter.js';
+export {CdpConnection} from '../cdp/cdpConnection.js';
```

Comment:
Let's not add a dependency on other modules for this file as the mapper only needs minimal CDP interface to work and we don't need to bring in the entire CDP client implementation (that is not used by Puppeteer)

Reply:
Should I delete `export {CdpConnection} from '../cdp/cdpConnection.js';` and instead just import it directly?

Reply:
I believe the src/bidiMapper should declare the interface it needs without the need to import the implementation interfaces

Reply:
I think that still imports the entire file and its dependencies. I think the duplication here is fine.

Reply:
@Lightning00Blade made the good point offline that if we do `import type` then this should likely not be the case. Updated again, PTAL.